WooCommerce/Classes/ViewRelated/Themes/ProfilerThemesPickerView.swift

```diff
@@ -0,0 +1,93 @@
+import SwiftUI
+import Kingfisher
+
+struct ProfilerThemesPickerView: View {
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+
+        ScrollView {
+            VStack(alignment: .leading) {
+                Spacer()
+                    .frame(height: Layout.contentVerticalSpacing)
+
+                Text(Localization.chooseThemeHeading)
+                    .bold()
+                    .largeTitleStyle()
+                    .padding(.horizontal, Layout.contentPadding)
+
+                Spacer()
+                    .frame(height: Layout.contentPadding)
+
+                Text(Localization.chooseThemeSubtitle)
+                    .subheadlineStyle()
+                    .padding(.horizontal, Layout.contentPadding)
+
+                Spacer()
+                    .frame(height: Layout.contentVerticalSpacing)
+
+                ThemesCarouselView(
+                    lastMessageHeading: Localization.lastMessageHeading,
+                    lastMessageContent: Localization.lastMessageContent
+                )
+
+                Spacer()
+            }
+        }
+        .toolbar {
+            ToolbarItem(placement: .navigationBarTrailing) {
+                Button(Localization.skipButtonTitle) {
+                    // TODO: Setup toolbar.
+                }
+            }
+        }
+        // Disables large title to avoid a large gap below the navigation bar.
+        .navigationBarTitleDisplayMode(.inline)
+    }
+}
+
+struct ProfilerThemesPickerView_Previews: PreviewProvider {
+    static var previews: some View {
+        ProfilerThemesPickerView()
+    }
+}
+
+private extension ProfilerThemesPickerView {
+    private enum Layout {
+        static let contentPadding: CGFloat = 16
+        static let contentVerticalSpacing: CGFloat = 40
+    }
+
+    private enum Localization {
+        static let skipButtonTitle = NSLocalizedString(
+            "ThemesPickerView.skipButtonTitle",
```

Comment:
Nit: we should follow the reverse domain name notation, which includes keeping the names in lower case (same for other keys below).

Reply:
Ah, you meant lowercase the first letter, right? I missed that. Thanks!

Reply:
https://github.com/woocommerce/woocommerce-ios/pull/11336/commits/74b07159e053d3b75b111128957b710e56c2d58c